Pipfile

```diff
@@ -32,6 +32,8 @@ wheel = "*"
 pipenv-setup = "*"
 pytest = "*"
 types-requests = "*"
+twine = "*"
```

Comment:
These two dependencies were "hidden" until now in `docs/requirements.txt` and in `make upgrade-dist-tools`. I think they belong here.

Reply:
I don't think we use mkdocstrins at all - it's a Mkdocs plugin that generates documentation based on the docstrings in the source code... and we don't have that, as far as I know. So probably this could be removed, right?

Reply:
I removed it, and GitHub CI runs through without errors. So I assume we really don't need it.